File: APICuponSmart/src/java/modelo/pojo/respuesta/RespuestaEstado.java
package modelo.pojo.respuesta;

import java.util.List;
import modelo.pojo.entidad.Estado;

public class RespuestaEstado extends Mensaje{
    private List<Estado> contenido;
    
    public List<Estado> getContenido(){
        return contenido;
    }
    
    public void setContenido(List<Estado> contenido){
        this.contenido = contenido;
    }
}

File: AppCuponSmart/app/src/main/java/uv/tc/appcuponsmart/data/model/respuesta/RespuestaEstado.kt
package uv.tc.appcuponsmart.data.model.respuesta

import uv.tc.appcuponsmart.data.model.entidad.Estado

class RespuestaEstado(
    error: Boolean,
    mensaje: String,
    var contenido: MutableList<Estado>? = null
): Mensaje(error, mensaje)